models/_sector/dex/trades/zksync/platforms/syncswap_zksync_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{
+    config(
+        schema = 'syncswap_zksync',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{% set syncswap_start_date = "2023-03-24" %}
+
+WITH 
+    -- All SyncSwap Pools
+    pools AS (
+        SELECT pool, token0, token1
+        FROM ref('syncswap_zksync_SyncSwapClassicPoolFactory_evt_PoolCreated') 
+        UNION ALL 
+        SELECT pool, token0, token1
+        FROM ref('syncswap_zksync_SyncSwapStablePoolFactory_evt_PoolCreated')
```

Comment:
These may be `source` instead of `ref`
```suggestion
        FROM {{ source('syncswap_zksync', 'SyncSwapClassicPoolFactory_evt_PoolCreated') }}
        UNION ALL 
        SELECT pool, token0, token1
        FROM {{ source('syncswap_zksync', 'SyncSwapStablePoolFactory_evt_PoolCreated') }}
```